src/Common/src/System/Security/Cryptography/KeySizeHelpers.cs

```diff
@@ -8,16 +8,11 @@
 
 namespace System.Security.Cryptography
 {
-    internal static class Helpers
+    internal static class KeySizeHelpers
     {
-        public static byte[] CloneByteArray(this byte[] src)
```

Comment:
Where did this CloneByteArray method go?  Is it just no longer used?

Reply:
Reused from Common/Internal/Security/Helpers.cs, which was already included in the other projects.